src/providers.ts

```diff
@@ -171,10 +186,10 @@ class LaunchDarklyCompletionItemProvider implements CompletionItemProvider {
 }
 
 const openFlagInBrowser = async (config: Configuration, flagKey: string, flagStore: FlagStore) => {
-	const { flag } = await flagStore.getFeatureFlag(flagKey);
+	const flag = await flagStore.getFeatureFlag(flagKey);
 
 	// Default to first environment
-	let env: FeatureFlagConfig = Object.values(flag.environments)[0];
+	let env = <FeatureFlagConfig>flag.environments[Object.keys(flag.environments)[0]];
```

Comment:
something like this might work
```suggestion
	let env = <FeatureFlagConfig>flag.environments.values().next().value;
```

Reply:
That one gives me an error:
```This expression is not callable.
  No constituent of type 'FeatureFlagConfig | FlagConfiguration' is callable.```

Reply:
Ah ok, we can do without this change then